### DISCUSSION TITLE
https://betsyninoska.github.io/   no found

### DISCUSSION THREAD

**Discussion by betsyninoska** [community/community#137371] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Error 
http://www.tavcode/   No se puede acceder a este sitioRevisa que no haya errores de ortografía en www.tavcode.
DNS_PROBE_FINISHED_NXDOMAIN

**Comment by LiteBrite82** [comment]
Which repo are you deploying from? can you add a screenshot of your pages settings?

**Reply by betsyninoska** [reply, marked as answer]
it's already working

![image](https://github.com/user-attachments/assets/f741ae34-c861-42e5-8040-a21a41dbf849)



thank you so much
